src/markdoc/partials/ray/quickstart.md

```diff
@@ -0,0 +1,69 @@
+
+# QuickStart
+
+This quickstart shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
+It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](/docs/creators/ray/setup-tutorial)
+
+
+## Install software
+
+{% partial file="ray/recommend-venv.md" /%}
+
+The first step is installing Ray on Golem. It will install Ray as a dependency.
+
+```bash
+# install ray-on-golem & ray (recommended within a clean virtual environment)
+pip3 install -U ray-on-golem
+```
+
+For now, you also need to download and install Golem node software representing you in the Golem network.
```

Comment:
do we have to? isn't yagna already included as a dependency?

Reply:
not yet, this will come in the next PR